packages/position-managers/src/config.test.ts

```diff
@@ -0,0 +1,89 @@
+import { chainNames } from '@pancakeswap/chains'
+import { Token } from '@pancakeswap/sdk'
+import difference from 'lodash/difference.js'
+import uniqBy from 'lodash/uniqBy.js'
+import { PublicClient, createPublicClient, http, parseAbiItem } from 'viem'
+import * as CHAINS from 'viem/chains'
+import { describe, expect, it } from 'vitest'
+import { SUPPORTED_CHAIN_IDS, VAULTS_CONFIG_BY_CHAIN } from './constants'
+
+const publicClient = Object.keys(VAULTS_CONFIG_BY_CHAIN).reduce((acc, chainId) => {
```

Comment:
Thanks, man, 
do you think we should merge this test with 
"apps/web/src/config/__tests__/vaults.test.ts"